src/webrtc/stats/summaryStatsReporter.ts

```diff
@@ -22,7 +24,7 @@ interface CallStatsReportSummaryCounter {
     totalAudio: number;
 }
 
-export class SummaryStatsReportGatherer {
+export class SummaryStatsReporter {
```

Comment:
This is a breaking change, as well as the file being renamed

Reply:
Well, rather than marking the PR as a breaking change, I think my concrete suggestion would be to keep the old names so that we don't force a major version bump for something so small